### ISSUE TITLE
Local Soundtrack.io Stuck on "No Playlists" Loop

### ISSUE THREAD

**Issue by ZachCase** [martindale/soundtrack.io#140]
@chrisinajar Do you know what is happening here, or what I need to do to get passed this? Martin mentioned that you touched that area, so maybe you'd know.

I am loading Soundtrack.io on my system locally with `mongod` and `redis-server` running, and when it gets to the `Room` portion of the `soundtrack.js` file it gets stuck on the loop that looks for Playlists. So it just prints `no playlists` to the console. The initial run of `soundtrack.js` created the room and asked for a restart, and when I ran it the second time it showed that 1 room was created but got stuck on the loop.

I've check the database to make sure everything was created that should have been and it all looked good. Were there supposed to be default playlist/tracks created as well?

**Comment by chrisinajar** [comment]
Yes, it's very very hard to break out of. I committed a few other small tweaks already to make this a little easier along with the bandcamp stuff. Here's what I remember...

* First, find the if (err) line that causes the infinte loop and let it break so that it can continue. Once you get it up, it will error and bitch and moan and freak out
* queue a bunch of songs anyway using manual posts.
* Restart it
* The queue will now work and continue to work until it's empty then it explodes again.

To get the empty queue to work, you must have queued songs at least 24 hours ago. You can get around this by editing that silly time function and making it start from tomorrow, and increase the window from 3 hours to 24.


Your best bet it to just get songs in to it by whatever bashing and mashing you can, then revert all your changes and wait until tomorrow.
